### PR TITLE
Adds doc describing how to install subman on Centos 7

### PR DESCRIPTION
**Pull request by cnsnyder**
This documents how to install subscription-manager on Centos 7.

### CODE REVIEW

File: docs/subscription-manager/how_to_install_subman_on_centos_7.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: How to install subscription-manager on Centos 7
+---
+{% include toc.md %}
+
+# How to install subscription-manager on Centos 7
+
+In order to install subscription-manager please do the following:
+
+1) Enable the subscription-manager copr repo
```
Comment (barnabycourt) [inline docs/subscription-manager/how_to_install_subman_on_centos_7.md]: Use 1. to indicate the beginning of a numbered list.

File: docs/subscription-manager/how_to_install_subman_on_centos_7.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+```bash
```
Comment (barnabycourt) [inline docs/subscription-manager/how_to_install_subman_on_centos_7.md]: Should be indented to match "Enabled.."  on line 10 in order for markdown to understand the list structure.

File: docs/subscription-manager/how_to_install_subman_on_centos_7.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+$ wget https://copr.fedoraproject.org/coprs/dgoodwin/subscription-manager/repo/epel-7/dgoodwin-subscription-manager-epel-7.repo -O /etc/yum.repos.d/dgoodwin-subscription-manager-epel-7.repo
+```
+This repo provides subscription-manager as well as python-rhsm packages.
+
+2) Install subscription-manager
```
Comment (barnabycourt) [inline docs/subscription-manager/how_to_install_subman_on_centos_7.md]: This line should start with "1." to continue the previously started list.

File: docs/subscription-manager/how_to_install_subman_on_centos_7.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+title: How to install subscription-manager on Centos 7
+---
```
Comment (barnabycourt) [inline docs/subscription-manager/how_to_install_subman_on_centos_7.md]: Missing "categories: ..." for this page.

**Comment by barnabycourt** [comment]
It seems odd to have a single Centos 7 page when we are building for multiple versions of Centos & Fedora. At a base this page should be a general installation page. Better would be help for all of our supported operating systems.

**Comment by cnsnyder** [comment]
Updated the document to be a bit more general (including info for f22, f23, epel 6 and epel 7).

**Comment by barnabycourt** [comment]
I like the update. Only nit pick, 'How to install subscription-manager from upstream' is a little long for a title. Perhaps just 'Installation'?

**Pull request merged**